.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 spec/functional/public/js/test-app.js
 npm-debug.log
 phantomjsdriver.log
+/.idea
```

Comment:
There's no need for this change. This is specific to your workflow and should probably be part of your global gitignore instead.